src/answer/answer.entity.ts

```diff
@@ -81,15 +78,9 @@ export class Answer {
       from: (value: string) => JSON.parse(value),
     },
   })
-  favoritedBy: string[];
+  favoritedBy: number[];
```

Comment:
I'm not sure whether it's the best way. In typical relational database, we avoid using array as type of a column for its huge performance lost. So this column definition, transforming and ussage in service.ts seems wierd to me.
Maybe in ORM this is also a supported and recommended way. Prisma is our soon-to-be-migrated ORM tool, and in Prisma you could investigate best and performance sound practices. I'm just to make sure you fully understand what you are doing here.

Reply:
See https://github.com/typeorm/typeorm/issues/2267